### DISCUSSION TITLE
How to move existing Swizzin in to new server?

### DISCUSSION THREAD

**Discussion by QuintusPL** [swizzin/swizzin#569] [Q&A + Help] [answered]
What is the easiest way to move everything from existing server to the new one?
I mean configurations users etc. Data can be moved easily but setting up everything from the beginning seems to be very time consuming as everything would have to be installed manually.

**Comment by flying-sausages** [comment]
Yeah this is one of the downsides, there is nothing that holds your hand for migration here _yet_. The easiest thing would be to install all the apps, create the users, stop all apps, overwrite their databases/configurations with the old files, and then start everything up again.

I'm planning on working on a feature like this sometime in the future but it would require specific per-app support. If you join me us in discord we can for sure help you get everything across right, and we can then document some of the app migration as well to make it easier for everyone in the future (and so I can have something to refer to when it gets to the code)

**Comment by flying-sausages** [comment, marked as answer]
We have since written a pretty in-depth guide on this which you can find here: https://swizzin.ltd/guides/migrating/

Let us know if you have anything which you think should be mentioned there.
